docs/pull_request_template.md

```diff
@@ -11,3 +11,6 @@ Optional.
 # Additional Context
 
 Optional. Feel free to add/modify additional headers as appropriate, e.g. "Refactorings", "Concerns".
+
+# Accessibility
+- [ ] Did you check our [checklist](https://www.notion.so/txi/Accessibility-checklist-224519b8cbea4a528acaaf5e91c7d09f)?
```

Comment:
This repo is open source - how about adding the checklist to the repo?  or including a shorter version here like:
```
- [ ] Did you scan for accessibility issues?
- [ ] Tables
- [ ] Inputs
- [ ] Color contrast
- [ ] Zoom
```

Reply:
Or maybe even simpler:
```suggestion
- [ ] Did you check our accessibility goal checklist?
```